backend/src/data_entry/api.rs

```diff
@@ -101,6 +103,12 @@ async fn polling_station_data_entry_claim(
     // Save the new data entry state
     polling_station_data_entries.upsert(id, &new_state).await?;
 
+    let data_entry = polling_station_data_entries.get_row(id).await?;
+
+    audit_service
+        .log(&AuditEvent::DataEntryClaimed(data_entry.into()), None)
+        .await?;
```

Comment:
I think it'd be better to create the audit event from `DataEntryStatus`, and also log the current status.

Reply:
We keep track of "progress" which could translate to a status. I could also keep a separate field with the status string.

We either way need to keep track of the polling station this relates to, that's why I used "PollingStationDataEntry".

Reply:
Added status label in 453bf5549efe69e129387b2fa21182f87b3ed719